example/styles.scss

```diff
@@ -51,13 +51,13 @@ a {
   z-index: 1;
   color: rgba(0,0,0,0);
   white-space: pre-wrap;
+  border: 1px solid rgba(0,0,0,0);
+  width: 100%;
```

Comment:
that is so the rendered container matches the border of the textbox
On Mar 6, 2016 2:44 PM, "Dean Sofer" <notifications@github.com> wrote:

> In example/styles.scss
> <https://github.com/angular-ui/ui-mention/pull/19#discussion_r55150211>:
>
> > @@ -51,13 +51,13 @@ a {
> >    z-index: 1;
> >    color: rgba(0,0,0,0);
> >    white-space: pre-wrap;
> > +  border: 1px solid rgba(0,0,0,0);
> > +  width: 100%;
>
> What are these doing?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/angular-ui/ui-mention/pull/19/files#r55150211>.
>